js/package.json

```diff
@@ -20,12 +20,14 @@
   "dependencies": {
     "@mantine/core": "7.10.2",
     "@mantine/hooks": "7.10.2",
+    "@tabler/icons": "^3.7.0",
```

Comment:
Are you even using this package? I don't see any references to it in your code.
If you decided you're not using it, you should remove it.